tests/test_introspection.py

```diff
@@ -0,0 +1,26 @@
+from graphql import graphql
+from graphql.utils.introspection_query import introspection_query
+
+from ariadne import make_executable_schema
+
+type_defs = """
+    type Query {
+        test: String
+        user(id: Int!): User
+        users: [User!]!
+    }
+
+    type User {
+        name: String
+        age: Int!
+        dateOfBirth: Date!
+    }
+
+    scalar Date
+"""
+
+
+def test_introspect_executable_schema():
```

Comment:
I would rename test to express its restricted scope to no-exception check.
Other thing is, we already use tests like those to integrate *ariadne* with *graph-core*, so we test expected output that depends on external library. Maybe, not saying that definitely, we could treat this test the same and check expected introspection response.